code/game/objects/items/weapons/tether.dm

```diff
@@ -0,0 +1,88 @@
+var/list/global/all_tethers = list()
+
+/obj/item/tethering_device
+	name = "tethering device"
+	desc = "A device used by explorers to keep track of partners by way of electro-tether."
+	desc_info = "Use in-hand to activate, must be on the same level and within fifteen tiles of another device to latch. Tethers are colour coded by distance."
+	icon = 'icons/obj/device.dmi'
+	icon_state = "locator"
+	w_class = ITEMSIZE_SMALL
+	slot_flags = SLOT_BELT
+	force = 1
+	var/active = FALSE
+	var/list/linked_tethers = list()
+	var/tether_range = 15
+	var/list/active_beams = list()
+
+/obj/item/tethering_device/Initialize(mapload, ...)
+	. = ..()
+	all_tethers += src
+
+/obj/item/tethering_device/Destroy()
+	STOP_PROCESSING(SSprocessing, src)
+	all_tethers -= src
+	deactivate()
```

Comment:
```suggestion
	deactivate()
	all_tethers -= src
```
Deactivate references all_tethers, and while your ordering is fine now, I think it will be safer against future changes to do it in this order instead.